ecs-cli/modules/commands/flags.go

```diff
@@ -77,6 +79,26 @@ const (
 	RoleFlag                                = "role"
 )
 
+// OptionalClusterClusterFlag inline selects a specific cluster configuration from the cluster config file
+func OptionalClusterConfigFlag() cli.Flag {
```

Comment:
What do you think of combining this with the existing `OptionalClusterFlag` because every place that allow this cluster flag will also need to cluster config flag.